QRCoderTests/Helpers/HelperFunctions.cs

```diff
@@ -2,10 +2,11 @@
 using System.Text;
 using System.IO;
 using System.Security.Cryptography;
+using System.Reflection;
 #if !NETCOREAPP1_1
```

Comment:
`#if !NETCOREAPP1_1` is this a candidate for `#if SYSTEM_DRAWING`?

Reply:
Curiously, no.  The PNG tests for the `net6.0` tfm utilize System.Drawing to perform hashing of the generated images.